insights/specs/insights_archive.py

```diff
@@ -59,11 +60,11 @@ class InsightsArchiveSpecs(Specs):
     grub1_config_perms = simple_file("insights_commands/ls_-l_.boot.grub.grub.conf")
     grub_config_perms = simple_file("insights_commands/ls_-l_.boot.grub2.grub.cfg")
     grubby_default_index = simple_file("insights_commands/grubby_--default-index")
+    gluster_v_info = simple_file("sos_commands/gluster/gluster_volume_info")
```

Comment:
For `insights_archive`, the simple_file should be `simple_file("insights_commands/gluster_volume_info")`